src/middleware/middleware.js

```diff
@@ -31,3 +37,42 @@ export const validateUserInput = (req, res, next) => {
   }
   return next();
 };
+
+export const validatePropertyInput = (req, res, next) => {
+  const {
+    image, title, address, landArea, noOfRoom, noOfBath,
+    noOfGarage, noOfStore, yearBuild, purpose
+  } = req.body;
+
+  if (!checkImageExtension(image)) {
+    res.status(400).send('Property image must be uploaded');
+  }
+  if (title === '') {
+    res.status(400).send('Property require title');
+  }
+  if (address === '') {
+    res.status(400).send('Property requires address');
+  }
+  if (landArea === '') {
+    res.status(400).send('Property land-area is required');
+  }
+  if (noOfRoom === '') {
+    res.status(400).send('no of rooms are required');
+  }
+  if (noOfBath === '') {
+    res.status(400).send('no of baths are required');
```

Comment:
You should send a message instead of a string, and your message should be very descriptive. Example: { message: "Number of baths are required" }